pkg/controllers/machineinventoryselector/bootstrap.go

```diff
@@ -146,6 +174,21 @@ func (h *handler) getBootstrapPlan(selector *v1beta1.MachineInventorySelector, i
 			{
 				CommonInstruction: applyinator.CommonInstruction{
 					Command: "/var/lib/rancher/bootstrap.sh",
+					// Ensure local plans will be enabled, this is required to ensure the local
+					// plan stopping elemental-system-agent is executed
+					Env: []string{
+						"CATTLE_LOCAL_ENABLED=true",
+					},
+				},
+			},
+			{
+				// Ensure the local plan can only be executed after bootstrapping script is done
+				CommonInstruction: applyinator.CommonInstruction{
+					Command: "bash",
+					Args: []string{
+						"-c",
+						"mv /var/lib/rancher/agent/plans/elemental-agent-stop.plan.skip /var/lib/rancher/agent/plans/elemental-agent-stop.plan",
```

Comment:
Im guessing this introduces a delay needed so if elemental and rancher agents overlap the rancher doesnt directly terminates the elemental one in case its still doing things? Did you see any overlap?